setup.py

```diff
@@ -76,7 +76,8 @@ def get_readme() -> str:
             "@gym_fixes#egg=stable-baselines3"
         ),
         "stable-baselines3>=1.4.0",
-        "sacred~=0.8.1",
+        # TODO(nora) switch back to PyPi once 0.8.3 makes it to release:
+        "sacred@git+https://github.com/IDSIA/sacred.git",
```

Comment:
do we want to use master v.s. the 0.8.3 release tag?

Reply:
Just changed to specify 0.8.3